newsfragments/289.feature.rst

```diff
@@ -0,0 +1 @@
+Make system tasks run in their own context defined at the ``trio.run`` call.
```

Comment:
How about:

```
Tasks created by :func:`~trio.hazmat.spawn_system_task` now no longer inherit the creator's :mod:`contextvars` context.
```

?

Reply:
I wasn't sure what to put for the note, so this sounds a lot better.